src/components/Amalgamation/AmalgamatingBusinesses.vue

```diff
@@ -229,7 +229,8 @@ export default class AmalgamatingBusinesses extends Mixins(AmalgamationMixin, Co
   legalName = null
   corpNumber = null
   isCan = false
-  isForeignBusinessValid = false
+  isForeignBusinessValid = null
```

Comment:
Can this be a Boolean (initialized to False)?

Reply:
I was thinking you'd initialize this to False..?

Reply:
I initialize it to null here because on line 269 I want to update the validation on the form when the jurisdiction changes, but only if the form is already in an active validation state (mainly because corp number is only required for Mras jurisdictions). I don't want to be showing or updating error messages unless the "Add" button is clicked. But once it is clicked, if invalid, I want to update the validation on the corp number field when the jurisdiction changes between Mras and non-Mras.

I could instead create a new variable named `showErrorsForeignBusiness`, and only show validation messages when it's true if that makes more sense.

Reply:
OK, so you do want the 3 values for a Boolean. Fair enough. Please either add comments explaining this, or implement a new variable. Up to you, with an eye to making it clear for a future dev 👍

Reply:
Great comments. Thanks! Resolving.